server/accounts.go

```diff
@@ -702,6 +702,9 @@ func (nl *NATSLatency) TotalTime() time.Duration {
 // ServiceLatency is the JSON message sent out in response to latency tracking for
 // exported services.
 type ServiceLatency struct {
+	Type           string        `json:"type"`
+	ID             string        `json:"id"`
+	Time           string        `json:"timestamp"`
```

Comment:
We have RequestStart which seems redundant a bit to this one, WDYT?

Reply:
ID is from the server sending this which can all be different etc. I am not sure we want to leak metadata to export acount owner on servers though. WDYT?

Reply:
> We have RequestStart which seems redundant a bit to this one, WDYT?

maybe, though start time would not be event generation time - by rights this should be different, but I agree we could probably drop one of the two.

Reply:
> ID is from the server sending this which can all be different etc. I am not sure we want to leak metadata to export acount owner on servers though. WDYT?

ages since I played with this, but which server would be publishing here? The one the client is connected to, it would be one event per request regardless right, need to play with this again to refresh my memory hehe 

So since I forgot how this works, I am not following your question, let me play again today/tomorrow and refresh my memory

Reply:
The requestor's server will generate. But point is I am trying to be aware of when we leak metadata and this could be between account owner for exported service and the operator of the server.

Reply:
isnt these messages published only within the account of the exporting service? There's no client identifying info here, so we're leaking things the account already knows effectively right, so I am not sure where the leak is?

Reply:
With a server ID or anything that could determine that we are leaking info to responder account on where the requestor is connected to (possibly).

Reply:
There's no server id here - or anything that identifies anything about the internals of the network? ID here is nuid random noise

Reply:
NUIDs are constructed to be very fast, so prefix remains the same for some period of time as suffix changes so if multiple requests have ID with same prefix they know its coming from same server. I know I am being pedantic.

Reply:
Hmm, ok, what do you suggest? I don't think its a big deal but fair, we could run it through a hash or something?

Reply:
In practice how is this field used?

Reply:
people will want to know when events are made, thats about it - many reason, you store them somehow and want to retrieve in order etc? These events might go to 3rd party systems, into streams etc, we cant know what people do - but they need a timestamp.

When the timestamp is kind of depends on the producer though, we might interpret timestamp here as request start time not event creation time, thats up to us.

Reply:
I meant the ID field, how is that used? Just as unique identifier?

Reply:
Just a unique identifier yes, carries no context but events need to have IDs.

So if you're concerned about it we can run this through a fast one way hash and that should sufficiently obfuscate it?

Reply:
ok will figure it our once the rewrite PR lands.